ml-algorithms/src/test/java/org/opensearch/ml/engine/algorithms/remote/HttpJsonConnectorExecutorTest.java

```diff
@@ -59,6 +66,133 @@ public void setUp() {
         MockitoAnnotations.openMocks(this);
     }
 
+    public void invokeRemoteModel_SuccessPath(String httpMethod) {
```

Comment:
remove "_".

Reply:
Can we not use "_" (underscore) in Java function names?

Reply:
@TrungBui59 In java, it's common practice to follow camel casing over than snake_case. Could you please address this too?